assignment5.js

```diff
@@ -0,0 +1,24 @@
+var fs = require('fs');
+
+var fileReader = function(fileName) {
+  return new Promise( (resolve, reject) => {
```

Comment:
Arrow function has the shortest syntax. It is also a new feature in ES6. I want to try something new so i use it. I can be used here also (var fileReader = (fileName) => { ... }  ).